Source/Assets/AppCenterEditorExtensions/Editor/Scripts/Utils/EdExLogger.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using UnityEngine;
+
+namespace AppCenterEditor
+{
+    class EdExLogger
```

Comment:
@annakocheshkova 
What about extracting the API to the Interface (in case we would like to change internal implementation in future)?

Reply:
@max-mironov why we need interface if now there's only one kind of logger and we can change it directly?

Reply:
Sure, always all stories starts with "for now we only do need this".
But what if e.g. we'd like to send logs to server in future? To use another custom implementation?
Programming to Interfaces is good, it promotes loose coupling. Inversion of Control is also a great thing in achieving this. 
Interfaces are flexible, the only thing you will need to change in future would be an implementation. Otherwise you will need to change all lines of code when you have reference to your logger. 

Also if you would like to add tests in future it would be easier to mock functionality in case we have an interface here. 

So I don't see any cons in using Interface there but there are potential pros in case of using IoC + Interface here. Just my opinion.